platform/podjs/src/browserPod.ts

```diff
@@ -97,26 +136,52 @@ class FileUrl {
         );
     }
 
+    /**
+     * Creates a new FileUrl object from the given data and fileName
+     * @param {string} data - The base URL of the file.
+     * @param {string} fileName - The name of the file.
+     * @returns A FileUrl object.
+     */
     static fromParts(data: string, fileName: string): FileUrl {
         const url = data + FileUrl.separator + fileName;
         return new FileUrl(url, data, fileName);
     }
-
-    constructor(
-        readonly url: string,
-        readonly data: string,
-        readonly fileName: string
-    ) {}
 }
 
-/* eslint-disable @typescript-eslint/no-unused-vars */
 class LocalStoragePolyOut implements PolyOut {
+    writeFile(
```

Comment:
Why does this appear as changed? Isn't this in `main` already?

Reply:
Just moved higher - I removed it by mistake as not-implemented and saw laters that it was needed.